package/yast2-registration.spec

```diff
@@ -58,6 +58,7 @@ BuildRequires:  yast2-slp >= 3.1.9
 # packager/product_patterns.rb
 BuildRequires:  yast2-packager >= 3.1.95
 BuildRequires:  yast2-update >= 3.1.36
+BuildRequires:  autoyast2-installation
```

Comment:
The only thing we need from that package is `Yast::AutoinstConfig.selected_product` which we mock here anyway.

I'd rather mock that YaST module completely and avoid this unnecessary build dependency. See [this code](https://github.com/yast/yast-packager/blob/master/test/test_helper.rb#L21-L28) how it's done in the packager.

Reply:
ok, will try